UnityProject/Assets/Scripts/Items/Tool/Welder.cs

```diff
@@ -102,6 +102,11 @@ public void OnSpawnServer(SpawnInfo info)
 		SyncIsOn(isOn, false);
 	}
 
+	public bool WillInteract(HandActivate interaction, NetworkSide side)
+	{
+		return DefaultWillInteract.Default(interaction, side);
```

Comment:
is this really the only check needed for this interaction?

Reply:
isn't it implemented by default if it's not specified?